README.md

```diff
@@ -1 +1,38 @@
-# my-to-do-list-
\ No newline at end of file
+# my-to-do-list-
+keep track of your tasks with this to do list 
+
+## DESCRIPTION
+build a simple HTML list of To Do tasks.
+
+## BUILT WITH
+- *Visual Studio Code*
+- *ES6*
+- *Linters for HTML/CSS/JS*
+- *JavaScript*
+- *CSS*
+- *Html*
+- *Webpack*
+
+## Getting Started
+
+**To get a local copy up and running follow these simple example steps**.
+
+### Prerequisites
+
+- You need to have a code editor to be able to run the code. VSCode is most preferred.
+
+### Setup
+
+- To clone the code press in the green button at the top right corner in this page
+- Select the method that you want to clone
```

Comment:
- [ ] Kindly update these steps and include how one can setup the webpack, eg: run npm i/ npm run build / npm start. There are so many technologies and frameworks, people may not know how to get started with particular this project 😉